eclipse-theia/theia
# plugin-ext: Add CodeActionTriggerKind enum

#### What it does
+ closes #9347
+ This commit adds the missing `CodeActionTriggerKind` enum.

#### How to test
+ Download and replace the default `TypeScript and JavaScript Language Features`  by `v1.55.0` from [download link](https://open-vsx.org/api/vscode/typescript-language-features/1.55.0/file/vscode.typescript-language-features-1.55.0.vsix).
+ Start theia
+ Open a `js` file, make changes.
+ Observe no errors from the console.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Co-Authored-by: @nihonium-cfx
Signed-off-by: Duc Nguyen <duc.a.nguyen@ericsson.com>
